src/neuroconv/datainterfaces/ophys/brukertiff/brukertiffdatainterface.py

```diff
@@ -1,12 +1,127 @@
+        return source_schema
+
+    @classmethod
+    def get_streams(
+        cls,
+        folder_path: FolderPathType,
+        plane_separation_type: Literal["contiguous", "disjoint"] = None,
+    ) -> dict:
+        from roiextractors import BrukerTiffMultiPlaneImagingExtractor
+
+        streams = BrukerTiffMultiPlaneImagingExtractor.get_streams(folder_path=folder_path)
+        channel_stream_name = streams["channel_streams"][0]
+        if plane_separation_type == "contiguous":
+            streams["plane_streams"].update({channel_stream_name: [streams["plane_streams"][channel_stream_name][0]]})
+        return streams
+
+    def __init__(
+        self,
+        folder_path: FolderPathType,
+        stream_name: Optional[str] = None,
+        plane_separation_type: Literal["contiguous", "disjoint"] = None,
+        verbose: bool = True,
+    ):
+        """
+        Initialize reading of TIFF files.
+
+        Parameters
+        ----------
+        folder_path : FolderPathType
+            The path to the folder that contains the Bruker TIF image files (.ome.tif) and configuration files (.xml, .env).
+        stream_name : str, optional
+            The name of the recording stream (e.g. 'Ch2').
+        plane_separation_type: {'contiguous', 'disjoint'}
+            Defines how to write volumetric imaging data. The default behavior is to assume the planes are contiguous,
+            and the imaging plane is a volume. Use 'disjoint' for writing them as a separate plane.
+        verbose : bool, default: True
+        """
+        self.streams = self.get_streams(folder_path=folder_path, plane_separation_type=plane_separation_type)
+        super().__init__(
+            folder_path=folder_path,
+            stream_name=stream_name,
+            verbose=verbose,
+        )
+        self._stream_name = self.imaging_extractor.stream_name.replace("_", "")
+        self._image_size = self.imaging_extractor.get_image_size()
+
+    def get_metadata(self) -> DeepDict:
+        metadata = super().get_metadata()
+
+        xml_metadata = self.imaging_extractor.xml_metadata
+        session_start_time = parse(xml_metadata["date"])
+        metadata["NWBFile"].update(session_start_time=session_start_time)
+
+        description = f"Version {xml_metadata['version']}"
+        device_name = "BrukerFluorescenceMicroscope"
+        metadata["Ophys"]["Device"][0].update(
+            name=device_name,
+            description=description,
+        )
+
+        imaging_plane_metadata = metadata["Ophys"]["ImagingPlane"][0]
+        imaging_plane_metadata.update(
+            device=device_name,
+            imaging_rate=self.imaging_extractor.get_sampling_frequency(),
+        )
+        two_photon_series_metadata = metadata["Ophys"]["TwoPhotonSeries"][0]
+        two_photon_series_metadata.update(
+            description="The volumetric imaging data acquired from the Bruker Two-Photon Microscope.",
+            unit="px",
```

Comment:
```suggestion
```

It's a fairly minor thing, but I'd say the units of 'pixels' are actually units of space; as previous in MICrONS they were used in reference to things like the estimated radius of pupil dilation w.r.t unknown physical units of space (say in 'cm') but rather in terms of known image space (number of pixels, 'px' which themselves would have unknown spatial equivalence 'n.a.')

Whereas the data of the two photon series is 'some amount of light' emitted from the sample and absorbed/detected by the camera over that pixels area. Also a very much unknown `n.a.` (the default of TwoPhotonSeries) until we start using Dmitri's measurement in terms of photon count per area over time